modules/shortcodes/twitter-timeline.php

```diff
@@ -19,6 +19,10 @@ function twitter_timeline_shortcode( $atts ) {
 
 	$output = '<a class="twitter-timeline"';
 
+	$partner = apply_filters( 'jetpack_twitter_partner_id', 'jetpack' );
```

Comment:
Once you've documented the filter in one place, you can refer to it like so:
`/** This filter is documented in modules/shortcodes/tweet.php */`